eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/ImageConverter.php

```diff
@@ -66,6 +66,7 @@ protected function createEmptyLegacyXml($contentMetaData)
         return $this->fillXml(
             array_merge(
                 array(
+                    'uri' => '',
```

Comment:
Note: while `__clone` fix makes this not needed, I still think we should ensure proper default value here.